components/Document/SubmissionDetails.tsx

```diff
@@ -45,8 +50,97 @@ function SubmissionDetails({
   const _twoDaysInMinutes = 2*24*60;
   return (
     <div className={css(styles.submittedBy)}>
-      <div className={css(styles.createdByContainer)}>
-        <AuthorAvatar author={authorProfile} size={avatarSize} trueSize />
+      <ResearchHubPopover
```

Comment:
This declarative usage of the component is ~60 lines long. The side of the entire `SubmissionDetails` component  Could we create a `UserPopoverWrapper` component?